src/scripts/oneNoteApiBase.ts

```diff
@@ -48,20 +48,21 @@ export class OneNoteApiBase {
 		return apiRootUrl + partialUrl;
 	}
 
-	private makeRequest(url: string, data?: XHRData, contentType?: string): Promise<ResponsePackage<any> | OneNoteApi.RequestError> {
+	private makeRequest(url: string, data?: XHRData, contentType?: string, verb?: string): Promise<ResponsePackage<any> | OneNoteApi.RequestError> {
```

Comment:
We should call 'verb' 'httpMethod' or similar. Should type this to an enum or string union.